cluster/saltbase/salt/kubelet/init.sls

```diff
@@ -31,19 +31,6 @@
     - mode: 400
     - makedirs: true
 
-#
```

Comment:
I was trying to clean it up since I thought everyone had moved off of it ages ago.